sample-app/src/components/InputDropdown.tsx

```diff
@@ -174,13 +177,13 @@ export default function InputDropdown({
             const value = evt.target.value;
             setLatestUserInput(value);
             onInputChange(value);
-            onInputFocus();
+            onInputFocus(value);
             setChildrenKey(childrenKey + 1);
             dispatch({ type: 'ShowSections' });
             setScreenReaderKey(screenReaderKey + 1);
           }}
           onClick={() => {
-            onInputFocus();
+            onInputFocus(inputValue);
```

Comment:
what is the difference between latestUserInput and inputValue?

Reply:
`inputValue` is the text in the input bar. `latestUserInput` is the last thing typed or queried by the user. the `inputValue` will change as the user navigates and focuses on different options, while `latestUserInput` will stay the same. `latestUserInput` is needed to reset the text in the input bar when the user navigates back to the input bar and no longer focuses on any option